mobile/src/bundles/common/components/selector/selector.tsx

```diff
@@ -123,7 +123,7 @@ const Selector = <T extends FieldValues>({
                     globalStyles.pl20,
                     globalStyles.width100,
                     isVisible && globalStyles.pb5,
-                    styles.dropdownButton,
```

Comment:
Does styles.dropdownButton is used anywhere else? If not we should remove it

Reply:
We use it, gives border to button